src/ripple/rpc/impl/Handler.cpp

```diff
@@ -151,6 +151,8 @@ Handler handlerArray[] {
     {   "unl_list",             byRef (&doUnlList),             Role::ADMIN,   NO_CONDITION     },
     {   "validation_create",    byRef (&doValidationCreate),    Role::ADMIN,   NO_CONDITION     },
     {   "validation_seed",      byRef (&doValidationSeed),      Role::ADMIN,   NO_CONDITION     },
+    {   "validator_lists",      byRef (&doValidatorLists),      Role::ADMIN,   NO_CONDITION     },
```

Comment:
What do you think about just calling this `validators`?

Reply:
Makse sense

Do you think `validator_sites` should become `validator_list_sites` to match https://github.com/ripple/rippled/blob/develop/doc/validators-example.txt#L26?

Reply:
Yeah, that's clearer